.github/workflows/release-v2.yml

```diff
@@ -288,6 +288,54 @@ jobs:
           JAVA_HOME=$JAVA_17 ./mvnw clean package -DskipTests -Dquarkus.container-image.build=true -Dquarkus.container-image.push=true -Dquarkus.container-image.registry=${{ secrets.ECR_REPOSITORY }} -Dquarkus.container-image.tag=${{needs.resolve-tag.outputs.release-tag}}
           cd ../
 
+  # signs all docker images with cosign
+  # skip whole job if we did not push images
+  sign-images:
+    name: Sign container images
+    needs: [resolve-tag, publish-docker]
+    if: ${{ !inputs.skipPublish }}
```

Comment:
I only now realize that `skipPublish` is bit ambiguous wrt Maven vs Docker publish?
It looks like it would mean skipping both Maven artifact and Docker image publish, right?

Reply:
Yes this is what I also specified in the description:

```
   skipPublish:
        description: 'Skip publishing docker images and JARs to OSSRH?'
```

Reply:
Ok sorry I missed that & was only trying to remember the logic. Sounds good.